providers/aws/aws_provider.py

```diff
@@ -200,3 +239,37 @@ def assume_role(audit_info: AWS_Audit_Info) -> dict:
 
     else:
         return assumed_credentials
+
+
+def get_organizations_metadata(assumed_credentials):
+    try:
+        organizations_client = client(
+            "organizations",
+            aws_access_key_id=assumed_credentials["Credentials"]["AccessKeyId"],
+            aws_secret_access_key=assumed_credentials["Credentials"]["SecretAccessKey"],
+            aws_session_token=assumed_credentials["Credentials"]["SessionToken"],
+        )
+        organizations_metadata = organizations_client.describe_account(
+            AccountId=current_audit_info.audited_account
+        )
+        list_tags_for_resource = organizations_client.list_tags_for_resource(
+            ResourceId=current_audit_info.audited_account
+        )
+        # Convert Tags dictionary to String
```

Comment:
I think every line from this point should be under the `else` clause.

Reply:
Yes, into the `try` statement only the calls to the apis, all the data processing in else clause